.github/workflows/build-and-test-arm7.yml

```diff
@@ -37,8 +37,7 @@ jobs:
       - name: cargo build for armv7-unknown-linux-gnueabihf
         run: |
           rustup target add armv7-unknown-linux-gnueabihf
-          cargo build --release --target armv7-unknown-linux-gnueabihf \
-            --no-default-features  --features bitdepth_8,bitdepth_16
+          cargo build --release --target armv7-unknown-linux-gnueabihf
```

Comment:
Do we still have separate tests with asm disabled?  If we don't, we should have one to catch any errors in the no-asm fallback code.